src/Utils/PhysCon.jl

```diff
@@ -12,13 +12,20 @@ export physcon
     σ::T = 5.67e-08;                 # Stefan-Boltzmann constant (W/m2/K4)
     mmdry::T = 28.97 / 1000;         # Molecular mass of dry air (kg/mol)
     mmh2o::T = 18.02 / 1000;         # Molecular mass of water (kg/mol)
-    Cpd::T = 1005;                   # Specific heat of dry air at constant pressure (J/kg/K)
-    Cpw::T = 1846;                   # Specific heat of water vapor at constant pressure (J/kg/K)
+    Cpd::T = 1005.;                  # Specific heat of dry air at constant pressure (J/kg/K)
+    Cpw::T = 1846.;                  # Specific heat of water vapor at constant pressure (J/kg/K)
+    Cpl::T = 4188.;                  # Specific heat of liquid water  at constant pressure at 15°C (J/kg/K)
```

Comment:
Can we move to using ClimaParameters for this (and add the missing parameters there)? This would make sure our code remains energetically consistent, and we use the same constants everywhere. @charleskawczynski can help (perhaps do this in a later PR).

Reply:
I think it'd be best to do this in a separate PR, since this pulls in a lot of new code. I'll open an issue.